src/System.Management.Automation/engine/Modules/ModuleUtils.cs

```diff
@@ -13,18 +13,30 @@ namespace System.Management.Automation.Internal
 {
     internal static class ModuleUtils
     {
+        // These are documented members FILE_ATTRIBUTE, they just have not yet been
+        // added to System.IO.FileAttributes yet.
+        private const int FILE_ATTRIBUTE_RECALL_ON_DATA_ACCESS = 0x400000;
+
+        private const int FILE_ATTRIBUTE_RECALL_ON_OPEN = 0x40000;
```

Comment:
I have not actually been able to produce a scenario where this attribute specifically appears (despite checking from multiple win32 APIs in case it only appeared in certain structs). I kept it in incase there's a different cloud provider that uses it, or if OneDrive starts to use it at some point.

Documentation for [FILE_ATTRIBUTE_RECALL_ON_OPEN](https://learn.microsoft.com/en-us/windows/win32/fileio/file-attribute-constants) reads:

> This attribute only appears in directory enumeration classes (`FILE_DIRECTORY_INFORMATION`, `FILE_BOTH_DIR_INFORMATION`, etc.). When this attribute is set, it means that the file or directory has no physical representation on the local system; the item is virtual. Opening the item will be more expensive than normal, e.g. it will cause at least some of it to be fetched from a remote store.

Reply:
If we can't even check it manually and even OneDrive doesn't use it, why is it PR?

Reply:
I kept it in incase there's a different cloud provider that uses it, or if OneDrive starts to use it at some point. Basically, documentation tell us to look for it, so I lean towards looking for it. Even if it happens not to be present in the specific use case we're looking to solve.

Reply:
You won't be the first anyway. 😸 
https://github.com/search?q=org%3Adotnet+FILE_ATTRIBUTE_RECALL_ON_OPEN&type=issues